test/LibraryManager.IntegrationTest/FileSaveRestoreTests.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class FileSaveRestoreTests : VisualStudioLibmanHostTest
+    {
+        ProjectTestExtension _webProject;
+        ProjectItemTestExtension _libManConfig;
+        const string _projectName = @"TestProjectCore20";
+
+        [TestInitialize()]
+        public void initialize()
+        {
+            _webProject = Solution.ProjectsRecursive[_projectName];
```

Comment:
>ProjectsRecursive [](start = 35, length = 17)

Is the project not in the root?  Usually we do `Solution[_projectName]`

Reply:
The project folder will be in the root of the solution folder.

---
In reply to: [194548753](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194548753) [](ancestors = 194548753)